example/src/main/java/org/nervos/ckb/example/MintSudtExample.java

```diff
@@ -0,0 +1,39 @@
+package org.nervos.ckb.example;
+
+import org.nervos.ckb.Network;
+import org.nervos.ckb.service.Api;
+import org.nervos.ckb.sign.TransactionSigner;
+import org.nervos.ckb.sign.TransactionWithScriptGroups;
+import org.nervos.ckb.transaction.SudtTransactionBuilder;
+import org.nervos.ckb.type.TransactionInput;
+import org.nervos.ckb.utils.Numeric;
+import org.nervos.indexer.InputIterator;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+public class MintSudtExample {
```

Comment:
suggest to rename `IssueSudtExample` since we are using `SudtTransactionBuilder.TransactionType.ISSUE`.